crypto/rsa/rsa_backend.c

```diff
@@ -7,10 +7,18 @@
  * https://www.openssl.org/source/license.html
  */
 
+#include <string.h>
 #include <openssl/core_names.h>
 #include <openssl/params.h>
+#include <openssl/evp.h>
+#include "internal/sizes.h"
+#include "internal/param_build_set.h"
 #include "crypto/rsa.h"
 
+#ifdef _WIN32
```

Comment:
Please do
#include "e_os.h" /* strcasecmp */

This header file is included in more than 50 other .c files. ifdef of a platform specific define is a way worse thing to start to block copy.

Reply:
[sigh] ok, fine